crates/extensions/c8y_config_manager/src/child_device.rs

```diff
@@ -4,16 +4,13 @@ use super::plugin_config::FileEntry;
 use c8y_api::smartrest::topic::C8yTopic;
 use log::error;
 use std::fs;
+use std::path::PathBuf;
 use std::time::Duration;
 use tedge_api::OperationStatus;
 use tedge_mqtt_ext::MqttMessage;
 use tedge_mqtt_ext::Topic;
 use tedge_mqtt_ext::TopicFilter;
 
-#[cfg(test)]
-pub const FILE_TRANSFER_ROOT_PATH: &str = "/tmp";
-#[cfg(not(test))]
-pub const FILE_TRANSFER_ROOT_PATH: &str = "/var/tedge/file-transfer";
```

Comment:
Lesson to be learned with this issue & PR.  Having different values for a `const` while under test or not is sign that something is wrong.